File: Source/frontend/frontend.Service/ILoginService.cs
using frontend.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace frontend.Service
{
    public interface ILoginService
    {
        List<Login> All();
        Login Find(int id);
        void Add(Login login);
        void Update(Login login);
        void Delete(int id);
    }
}

File: source/back-end/src/main/java/be/pxl/backend/service/ILoginService.java
package be.pxl.backend.service;

import java.util.List;
import be.pxl.backend.entity.Login;

public interface ILoginService {

	public Login find(int id);
	
	public List<Login> all();
	
	public void persist(Login login);
	
	public void delete(int id);
	
}
